server/http/helloapp/bindings.go

```diff
@@ -0,0 +1,60 @@
+package helloapp
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/api"
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+)
+
+// Install function metadata is not necessary, but fillint it out (minimally)
+// for demo purposes. Install does not bind to any locations, it's Expand is
+// pre-determined by the server.
+func (h *helloapp) handleBindings(w http.ResponseWriter, req *http.Request, claims *apps.JWTClaims, cc *api.Context) (int, error) {
+	call := h.makeCall(PathMessage)
+	modal := *call
+	modal.AsModal = true
+
+	httputils.WriteJSON(w,
+		[]*api.Binding{
+			{
+				LocationID: api.LocationCommand,
+				Bindings: []*api.Binding{
+					{
+						LocationID:  "message",
+						Hint:        "[--user] message",
+						Description: "send a message to a user",
+						Call:        call,
+					}, {
```

Comment:
If I understand correctly, what's going on here is we are telling the client:

"Load this sub command. If `/hello message` it is typed, fetch the form from the hello app from the `/message` endpoint." Which is the same endpoint that is used for the modal defined below.

Reply:
Discussed offline, basically yes, but still TBD on the details. Looking forward to the first implementation of Autocomplete or Modal to tie it all together.